kpm/lib/kpm/base_artifact.rb

```diff
@@ -173,9 +172,9 @@ def artifact_info(logger, coordinate_map, overrides={}, ssl_verify=true)
         coordinates = KPM::Coordinates.build_coordinates(coordinate_map)
         begin
           nexus_info = nexus_remote(overrides, ssl_verify).get_artifact_info(coordinates)
-        rescue NexusCli::ArtifactMalformedException => e
-          raise NexusCli::NexusCliError.new("Invalid coordinates #{coordinate_map}")
-        rescue NexusCli::NexusCliError => e
+        rescue KPM::NexusFacade::ArtifactMalformedException => e
```

Comment:
One related Travis test is failing (https://travis-ci.org/killbill/killbill-cloud/jobs/254647975), could you check if it's a regression?

```
  1) KPM::KillbillPluginArtifact should be able to list versions
     Failure/Error: versions = KPM::KillbillPluginArtifact.versions
     KPM::NexusFacade::ArtifactMalformedException:
       KPM::NexusFacade::ArtifactMalformedException
     # ./lib/kpm/nexus_helper/nexus_api_calls_v2.rb:115:in `parse_coordinates'
     # ./lib/kpm/nexus_helper/nexus_api_calls_v2.rb:146:in `get_query_params'
     # ./lib/kpm/nexus_helper/nexus_api_calls_v2.rb:160:in `get_response'
     # ./lib/kpm/nexus_helper/nexus_api_calls_v2.rb:51:in `search_for_artifacts'
     # ./lib/kpm/nexus_helper/actions.rb:31:in `search_for_artifacts'
     # ./lib/kpm/killbill_plugin_artifact.rb:18:in `block in versions'
     # ./lib/kpm/killbill_plugin_artifact.rb:17:in `each'
     # ./lib/kpm/killbill_plugin_artifact.rb:17:in `versions'
     # ./spec/kpm/remote/killbill_plugin_artifact_spec.rb:75:in `block (2 levels) in <top (required)>'
```

Don't worry about the other `migrations` tests failing, these are expected to fail on PR because the GitHub token isn't passed to the pull request -- see https://docs.travis-ci.com/user/environment-variables/#Defining-encrypted-variables-in-.travis.yml.